WrightTools/units.py

```diff
@@ -111,12 +116,15 @@ def converter(val, current_unit, destination_unit):
     number
         Converted value.
     """
-    try:
-        val = ureg.Quantity(val, current_unit).to(destination_unit).magnitude
-    except (pint.errors.DimensionalityError, pint.errors.UndefinedUnitError, AttributeError):
+    if (current_unit == None) and (destination_unit == None):
```

Comment:
please use `is None` instead of `== None` (this is more efficient and standard style (see code scanning results)